dashboard/db/migrate/20201117160845_create_user_ml_models.rb

```diff
@@ -0,0 +1,20 @@
+class CreateUserMlModels < ActiveRecord::Migration[5.2]
+  def up
+    create_table :user_ml_models do |t|
+      t.integer :user_id
+      t.string :model_id
+      t.string :name
+      t.datetime :deleted_at
+      t.datetime :purged_at
+
+      t.timestamps
+    end
+    add_index :user_ml_models, :user_id
+    add_index :user_ml_models, :model_id
+    execute "ALTER TABLE user_ml_models CONVERT TO CHARACTER SET utf8 COLLATE utf8_unicode_ci"
+  end
```

Comment:
@sureshc like this?

Reply:
Yes!! Sorry for the inconvenience. We will soon resolve this character set discrepancy and this extra logic won’t be needed.